content/documentation/pages/3-stream-developer-guides/1-getting-started/1-stream.md

```diff
@@ -157,59 +184,8 @@ Now you can delete the stream you created. To do so:
 
 ## Updating and Rolling back a Stream
 
-This information can be found in the [Continuous Delivery Basics Guide](%currentpath%/stream-developer-guides/continuous-delivery/cd-basics)
+This information can be found in the [Continuous Delivery](%currentpath%/stream-developer-guides/continuous-delivery) guide.
```

Comment:
%currentpath% is not being replaced.   So the link is invalid.

Reply:
I'm checking with @oodamien on this one